dashboard/test/controllers/registrations_controller/new_test.rb

```diff
@@ -22,21 +22,21 @@ class NewTest < ActionDispatch::IntegrationTest
     #
     # NEW SIGNUP FLOW
     #
-    test 'renders finish_sign_up partial registration is in progress in new flow' do
-      SignUpTracking.expects(:new_sign_up_experience?).returns(true).twice
-      PartialRegistration.expects(:in_progress?).returns(true).twice
-      User.expects(:new_with_session).returns(build(:user))
+    test 'renders new signup form if partial registration is not in progress in new flow' do
```

Comment:
The rest of the diff in this file is very misleading... I re-ordered the tests into a more logical order and updated some `.expects` statements to `.stubs` (I didn't actually care how many times the method was called, and had to keep updating my expect-statements, so it made more sense for them to be stubs).